app/controllers/users_controller.rb

```diff
@@ -1,3 +1,24 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find(params[:id])
```

Comment:
phân biệt find và find_by? ở đây nên sử dụng cái nào?
Thêm xử lý cho trường hợp ko tìm thấy user nữa ạ

Reply:
User.find(1): là method tìm user theo id là 1, nếu id không có trong table sẽ báo lỗi.
User.find_by(id: 1): là method tìm user theo trường id với giá trị là 1. Có thể chỉ định tìm theo trường khác ví dụ: User.find_by(name: "Your name"). Nếu trong table không có thì sẽ trả về nil.
Trường hợp này nên dùng find(params[:id]). Để có thể try catch.

Reply:
Thế dùng find_by rồi kiểm tra nil được ko ạ? để raise exception và trả về nil thì cái nào an toàn hơn cho app ạ?

Reply:
ah. vẫn được. khi hồi làm vậy mà rubocop báo convention nên chuyển qua try catch mà vẫn bị convention. keke